### PR TITLE
fix common misspells

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux vieux@docker.com

### CODE REVIEW

**Comment by vieux** [comment]
ping @moxiegirl @thaJeztah

**Comment by jfrazelle** [comment]
how did you find all these like a parser

**Comment by jfrazelle** [comment]
LGTM

**Comment by vieux** [comment]
@jfrazelle yup, https://github.com/client9/misspell, but it's not perfect, it tried to rewrite some stuff in seccomp :)

**Comment by jfrazelle** [comment]
sweeeet

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -75,7 +75,7 @@ var (
 		"trusting",
 	}
 
-	// Docker, starting from 0.7.x, generates names from notable scientists and hackers.
+	// Docker, starting from 0.7.x, generates names from notable scientistts and hackers.
```
Comment (tonistiigi) [inline pkg/namesgenerator/names-generator.go]: @vieux ^

Comment (vieux) [inline pkg/namesgenerator/names-generator.go]: ah thanks

File: daemon/execdriver/driver_windows.go [hunk 2]
```diff
@@ -49,7 +49,7 @@ type Command struct {
 
 	// Fields below here are platform specific
 
-	FirstStart  bool     `json:"first_start"`  // Optimisation for first boot of Windows
+	FirstStart  bool     `json:"first_start"`  // Optimization for first boot of Windows
```
Comment (jhowardmsft) [inline daemon/execdriver/driver_windows.go]: Hey this was correct for the Brit who wrote this! :smiling_imp:

**Comment by thaJeztah** [comment]
Darn! you beat me to it, haha https://github.com/docker/docker/pull/20252

File: daemon/execdriver/driver_windows.go (hunk 2 above)
Comment (calavera) [inline daemon/execdriver/driver_windows.go]: I knew this would happen! :laughing:

**Comment by MHBauer** [comment]
All corrections LGTM.

**Comment by icecrime** [comment]
image

**Comment by MHBauer** [comment]
@jhowardmsft see, windows doesn't like the new spelling either. It is quite fast, though...

**Pull request merged**